rel/files/app.config

```diff
@@ -6,7 +6,7 @@
     {log_root, "{{mongooseim_log_dir}}"},
     {crash_log, "crash.log"},
     {handlers, [
-        {lager_console_backend, [info, {lager_default_formatter,[{eol, "\r\n"}]}]},
+        {lager_console_backend, [{level, info}]},
```

Comment:
because currently lager_console_backend crashes just after start and we don't have logs.
```erlang
2018-04-30 11:45:07.329 [error] <0.205.0>@lager_handler_watcher:123 
Lager fatally failed to install handler lager_console_backend into lager_event, NOT retrying: {bad_console_config, info}
```

Reply:
will put in a separate PR.